libs/service-portal/core/src/components/UserInfoLine/UserInfoLine.tsx

```diff
@@ -9,7 +9,9 @@ import {
 } from '@island.is/island-ui/core'
 import { useLocale } from '@island.is/localization'
 import { MessageDescriptor } from 'react-intl'
-import { Link } from 'react-router-dom'
+import { Link, useLocation } from 'react-router-dom'
+import { ServicePortalOutboundLink } from '@island.is/plausible'
```

Comment:
Shouldn't the function name be lowercase?

Reply:
You are correct